week8/JungSangHyun/src/pages/Cart/Cart.tsx

```diff
@@ -0,0 +1,145 @@
+import { useNavigate } from "react-router-dom";
+import useCartStore from "../../stores/useCartStore";
+import useStoreStore from "../../stores/useStoreStore";
+
+const Cart = () => {
+  const navigate = useNavigate();
+  const { storeId, menus, removeMenu, clearCart } = useCartStore();
```

Comment:
store에서 가져올 때 한번에 가져오지 않고 따로따로 가져오거나 useShallow를 쓰면 성능에 더 좋습니다.
```suggestion
const storeId = useCartStore(state => state.storeId);
const menus = useCartStore(state => state.menus);
// ...
```
```suggestion
const { storeId, menus} = useCartStore(
  useShallow((state) => ({ storeId: state.storeId, menus: state.menus})),
)
```
https://github.com/pmndrs/zustand?tab=readme-ov-file#recipes
요기에 readme에 recipes쪽 보면 관련된 내용이 있습니담